design/dialogs/brianna1.md

```diff
@@ -266,7 +266,7 @@ Some seconds pass and we just hear Marjorie crying and sobbing. Then:
 
 > [marjorie](../characters/marjorie.md)
 > 
-> Okay I... I freaked. It was all too much, you know?
+> Okay I... I freaked out. It was all too much, you know?
```

Comment:
I actually meant "freaked" here, which means "heavily startled" according to my dictionary.

Reply:
OK. I don't think there's any real difference. According to Google:
_Freaked: Behave or cause to behave in a wild and irrational way.
Freak out: A wildly irrational reaction or spell of behaviour._
I just changed it since "freaked" is so much more uncommon. "I was freaked" _not_ followed by "out" only matches 8 % of the hits on Google. But it doesn't matter, I guess.